src/dispatch/plugins/dispatch_slack/case/messages.py

```diff
@@ -124,70 +125,36 @@ def create_case_message(case: Case, channel_id: str):
 def create_signal_messages(case: Case) -> List[Message]:
     """Creates the signal instance message."""
     messages = []
+
     for instance in case.signal_instances:
+        button_metadata = SubjectMetadata(
+            type="signalInstance",
+            organization_slug=case.project.organization.slug,
+            id=str(instance.id),
+            project_id=case.project.id,
+            channel_id=case.conversation.channel_id,
+        ).json()
+
         signal_metadata_blocks = [
             Section(
-                text="*Signal Details*",
+                text=f"*Signal Entities* - {instance.id}",
                 accessory=Button(
-                    text="View",
-                    url=f"{DISPATCH_UI_URL}/{case.project.organization.slug}/signals/{instance.id}",
```

Comment:
Should we keep a link to the Case UI somewhere?

Reply:
The top-level case message has a link to the Case UI.